peter-piper.js

```diff
@@ -22,4 +22,11 @@ var counter = 0;
 // TODO: Count the number of times an uppercase P or lowercase p character
```

Comment:
This "TODO" is no longer necessary. Either remove the word "TODO" or remove the comment entirely.